docs-client/src/containers/MethodPage/DebugPage.tsx

```diff
@@ -161,7 +179,7 @@ const DebugPage: React.FunctionComponent<Props> = ({
     if (useRequestBody) {
       if (urlParams.has('request_body')) {
         urlRequestBody = jsonPrettify(urlParams.get('request_body')!);
-        scrollToDebugForm();
+        setDebugFormIsOpen(true);
```

Comment:
In addition to `request_body`, `headers`, `endpoint_path` and `queries` could be set to `urlParams`. How about showing the debug form if `urlParams` has values?